QModManager/API/SMLHelper/MonoBehaviours/Fixer.cs

```diff
@@ -0,0 +1,55 @@
+namespace QModManager.API.SMLHelper.MonoBehaviours
+{
+    using UnityEngine;
+    using System.Reflection;
+    using Logger = QModManager.Utility.Logger;
+    using Harmony;
+
+    public class Fixer : MonoBehaviour, IProtoEventListener
```

Comment:
Is this meant for public usage?

Reply:
I don't believe so, given that we manage adding this component ourselves.  
It could be `internal`, since I don't think we've ever given any guidance to modders to use it directly.